build.sh

```diff
@@ -0,0 +1,17 @@
+# Build the images
```

Comment:
I think it would be good to have a comment saying that this build is for a local Docker build and not for other types of deployments. WDYT?

Reply:
Although this was written in a doc file, it would also be a good idea to have it written in the file as well.